roles/monitor/templates/monitor.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+if test -f {{ ansible_env.HOME }}/ocluster.log ; then
+	kc=$(tail {{ ansible_env.HOME }}/ocluster.log | grep pkill | wc -l)
+	if [ $kc -eq 10 ] ; then
+		rm -f {{ ansible_env.HOME }}/ocluster.log
```

Comment:
Possibly, just move it to `ocluster.old` to avoid the burden of needing to manage a build up of old logs files.

Reply:
I think that should work, tho I am slightly worried it could mask problems that lead to a rebooting loop. It'd be much easier to troubleshoot that kind of problem if we had a trail rather than just the last log. I'm fine with just going with old for now, tho if we have a more targeted way of monitoring, as I've suggested below, the whole issue of having to move the logs would go away. M